plugins/netmiko/requirements.txt

```diff
@@ -2,4 +2,5 @@
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
 netmiko==3.3.3
-paramiko==2.7.2
+paramiko==3.4.0
+setuptools==70.0.0
```

Comment:
With SDK 6.0.1, we can remove this requirement.

Reply:
I'm going to keep this included to be consistent with the other recent snyk fixes